src/v/datalake/translation/tests/scheduler_fixture.cc

```diff
@@ -234,6 +234,7 @@ translation_status mock_translator::status() const {
     status.next_checkpoint_deadline = _next_checkpoint;
     if (_translation_state) {
         status.memory_bytes_reserved = total_bytes_reserved();
+        status.disk_bytes_flushed = total_bytes_reserved();
```

Comment:
adding for completeness of filling in the status{} structure. there aren't any top-level tests dedicated to status, but clearly it is just a conduit for passing up data from the translation context which is tested.